### ISSUE TITLE
dosent go into active state (pipewire)

### ISSUE THREAD

**Issue by Ffuifffghbfssdkkc** [noisetorch/NoiseTorch#410]
**Describe the bug**

says noise torch inactive, will not set active state

**To Reproduce**

open noise torch
configure input device
click set avtive
wait while stuff happens
nothing happens

**Expected behavior**

set active state and noise torch funtion

**Screenshots**

**Logs**
If applicable, add the link to a pastebin with the output of `noisetorch -log` and `pactl list short`.

https://pastebin.com/0abswT9j

**Desktop (please complete the following information):**
 - Distribution [e.g. Ubuntu 22.04, Arch]: 
 - DE [e.g. Plasma 5.24.5, GNOME 42.0]: 
 - Pulseaudio/Pipewire Version: 
 - NoiseTorch-ng Version [e.g. v0.12.0]:

arch lts
kde plasma
pipewire
https://pastebin.com/0abswT9j

**Additional context**

**Comment by Ffuifffghbfssdkkc** [comment]
note, i found a temp fix. replace pipewire-pulse with pulse audio

**Comment by TheDukeofErl** [comment]
Odd, I don't have any issues on Arch with pipewire/pipewire-pulse/wireplumper. Can you please provide the output of `pactl list short`? Also, where are you getting your build from? Building from source?

**Comment by Ffuifffghbfssdkkc** [comment]
Hi, I built it from an air page. I'm not sure how to compile from source. Not at my pc atm but I'll explain what happened since my last post. After I shutdown my pc and turned it on my audio cut out on mic and headphones. After a lot of fucking about I found that for some reason pipewire had become masked in an odd location misspelled. I am confused and maybe something weird to do with the makepkg but now I fully removed the mask and pulse audio it's all working fine. I also switched from pipewire media session to wire plumber. I'll double check it's working correctly in the morning but not at my pc atm. I'll keep you updated I just didn't check properly yet

**Comment by TheDukeofErl** [comment]
Understood, thanks for the update. For what it's worth, all of those are also user services not system, so sometimes if really odd things are happening `systemctl restart --user pipewire pipewire-pulse wireplumber` can help it calm down a little bit. Likewise, if you want status, it's still a user service.

I haven't had issues with the AUR package, though being lazy I generally use https://github.com/Jguer/yay to build/install. If you're manually installing things via the PKGBUILD files it may be worth using an AUR helper.

**Comment by Ffuifffghbfssdkkc** [comment]
Oh for some reason I never liked using yay as much as just manually doing the makepkg,I just do makepkg -si --clean and call it a day lol. Think checking the aur page and reading some comments is preferable to me before downloading. But yeah I did try reinstalling all audio servers and rebooting etc but until I tracked down and unmasked this one pipewire service it wouldn't work at all. Maybe some mistake on my end I'm really not sure why that happened lol
